app/views/govuk_publishing_components/components/_table.html.erb

```diff
@@ -2,46 +2,63 @@
   caption ||= nil
   head ||= []
   rows ||= []
-  first_cell_is_header ||=false
+  first_cell_is_header ||= false
   caption_classes ||= nil
-  sortable ||=false
+  sortable ||= false
+  filterable ||= false
 %>
 
-<%= GovukPublishingComponents::AppHelpers::TableHelper.helper(self, caption, {
-    sortable: sortable,
-    caption_classes: caption_classes
-  }) do |t| %>
+<div <% if filterable %> data-module="table-filter" <% end %> >
+  <% if filterable %>
+    <div class="govuk-table-filter">
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Search"
+        },
```

Comment:
Does `Search` adequately convey the meaning of this control?

Also `Search` and `Filter` seem to be used interchangeably in a few places across this PR, but I think users might consider them to be different things and therefore this could be confusing.

Reply:
This has been updated for consistency. I think though that the actual text that should be displayed and the style in which it is rendered is more a design specialism.